aws/logs_monitoring/template.yaml

```diff
@@ -391,6 +398,10 @@ Conditions:
     Fn::Equals:
       - Ref: DdForwardLog
       - false
+  SetDdStepFunctionTraceEnabled:
+    Fn::Equals:
+      - Ref: DdStepFunctionTraceEnabled
+      - true
```

Comment:
Confused if this should be true or false

`DdForwardLog` case above sets it to false but doesn't that mean the variable will be true if the parameter is set to false?

Reply:
These parameters are shown to customers when they are deploying the DD-Forwarder (as a Lambda).

I think the workflow is Cloud Formation Parameters (Default false for customers) => Forwarder env variables
Then, during the runtime, read this env variable and handle logs accordingly.
 
aws cloudformation config when [install new Forwarder ](https://docs.datadoghq.com/logs/guide/forwarder/?tab=cloudformation)
<img width="1375" alt="Screenshot 2024-07-31 at 11 27 05 AM" src="https://github.com/user-attachments/assets/5911380f-642b-4489-b9cd-08b5ea84e46b">
// aws console
<img width="1367" alt="Screenshot 2024-07-31 at 11 29 13 AM" src="https://github.com/user-attachments/assets/fec0ab06-4fcc-4286-ab2b-c0b681faeef6">

Above is my understanding, you may double check with @kimi-p to verify it.